src/main/scala/no/ndla/imageapi/ImageApiProperties.scala

```diff
@@ -42,6 +42,9 @@ object ImageApiProperties extends LazyLogging {
   val MetaInitialConnections = 3
   val MetaMaxConnections = 20
   val Environment = propOrElse("NDLA_ENVIRONMENT", "local")
+  val (redDBSource, cmDBSource) = ("red", "cm")
+  val ImageImportSource = if (Environment == "prod") cmDBSource else redDBSource
```

Comment:
Enig i defaulten, men burde ikke dette kunne styres "utenfra"? Ref. ønsket om å kunne teste cm-data på et senere tidspunkt...

Reply:
Jeg kan legge på parametre til importen slik at vi styrer dette via query parametre utenfra. Det som kanskje kan skje da er at vi ender opp med miksa innhold i miljøer. Altså noe fra red og noe fra cm. Det er kanskje et større problem i article-api enn i image-api

Reply:
Enig, men trodde du hadde åpnet for det andre steder... Slik at det var dette som ville skapt trøbbel (dersom du spesifiserte cm i andre api'er, men her brukte red pga. hardkodingen). Hvis dette følger standarden vi har ellers, foreslår jeg å la det være. Særs enkelt å utvide den dagen vi trenger å hente data fra cm i andre miljøer...

Reply:
La inn samme logikk med samme tankegang i de andre APIene også. Importkilde er bestemt ut i fra miljø og det er (foreløpig) ikke mulig og overstyre fra en ekstern klient. Det er kun til migration-api det kan oversyres hvilke database den skal gå mot :)